### PR TITLE
Add ex-navigator to UI list

### PR DESCRIPTION
**Pull request by simplyianm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -299,7 +300,7 @@ Components and native modules. For more search [React Parts](http://react.parts/
 - [react-native-app-info ★14](https://github.com/Iragne/react-native-app-info) - React Native app info and version
 - [react-native-barcode-scanner ★29](https://github.com/lifuzu/ReactNativeBarcodeScanner) - Barcode scanner for React Native
 - [react-native-barcodescanner ★29](https://github.com/ideacreation/react-native-barcodescanner) - A barcode scanner component for react native
-- [react-native-battery-status ★1](https://github.com/remobile/react-native-battery-status) - A battery-status for react-native
+- [react-native-battery-status ★1](https://github.com/remobile/react-native-battery-status) - A battery-status for react-nativen
```
Comment (olivierlesnicki) [inline README.md]: nativen???

**Comment by jondot** [comment]
Great!

**Comment by olivierlesnicki** [comment]
@jondot but hat is react-nativen?

**Comment by jondot** [comment]
Might have originally been a typo. I'll fix that, thanks for catching it :+1:

**Comment by simplyianm** [comment]
Oops

**Pull request merged**